docs/tutorials/kafka-migration.rst

```diff
@@ -0,0 +1,368 @@
+
+The data migration will be performed using :doc:`MirrorMaker 2 </docs/products/kafka/kafka-mirrormaker>`, a fully managed distributed Apache Kafka® data replication utility.
+
+
+.. mermaid::
+
+    graph LR;
+
+        id1(Source Apache Kafka cluster)-- source alias -->id2(MirrorMaker 2);
+        id2-- sink alias -->id4(Target Apache Kafka cluster);
+
+.. Warning:: 
+
+  MirrorMaker 2 provides an **asynchronous** replication across clusters. Therefore, to perform a successful migration avoiding data loss, you need to ensure that the replication lag is 0 before pointing the new producers and consumers to the target Kafka environment. More information in the :ref:`monitor the MirrorMaker 2 replication flow lag <tutorial_kafka_migration_replication_flow_lag>` section.
+
+
+Prerequisites
+-------------
+
+The exact migration steps can vary, depending on the peculiar usage of the existing source system. This tutorial however assumes the following pre-requisites:
+
+* **Clients compatibility**: Any clients interacting with Apache Kafka must be compatible with the version chosen as migration target.
+
+  .. Note::
+
+    This guide doesn't include any steps to migrate clients versions.
+
+* Available **Apache Kafka configurations**: the custom settings needed in the source system, should be available as parameters in the target system. 
+
+  .. Note::
+
+    You can find a list of Aiven for Apache Kafka advanced parameters in the :doc:`related documentation </docs/products/kafka/reference/advanced-params>`.
+
+* Available **Apache Kafka Connect connectors**: if you're planning to migrate a Kafka Connect cluster, review the list of :doc:`supported connectors </docs/products/kafka/kafka-connect>` or check out how to :doc:`bring your own Kafka Connect cluster </docs/products/kafka/kafka-connect/howto/bring-your-own-kafka-connect-cluster>`. 
+
+* **Connectivity**: the source and target clusters need to be connected via MirrorMaker 2 for the data migration, therefore proper connection needs to be established. Moreover, you need to ensure all the clients reading or writing to Apache Kafka can connect to the brokers as well as any self hosted Kafka connect cluster. 
+
+  .. Warning::
+
+    As part of the connectivity check you also need to ensure:
+        
+    * That the **connection bandwidth** is enough to sync the data between environments.
+    * That the needed networking setups are available in the target Apache Kafka cluster, some examples on how to define custom networking setups on Aiven:
+        * :doc:`Firewalls/security groups </docs/platform/concepts/cloud-security>`.
+        * :doc:`VPC peering </docs/platform/howto/manage-vpc-peering>`.
+        * :doc:`Privatelink </docs/platform/howto/use-aws-privatelinks>`.
+
+
+Create an Apache Kafka integration endpoint
```

Comment:
Hi Celeste! 
Good question. I believe this tutorial is more "high level" and requires the user to have an external Kafka cluster that is accessible by Aiven. We, on purpose, tried to avoid any specific vendor and go with more of a high level approach.

I'll add a line to drive people to [here](https://github.com/apache/kafka/blob/trunk/connect/mirror/README.md) to say if you're using standalone MM2, you can define sources and sinks in the MM2 config file

Reply:
I think that's a good idea (that won't take an absolute ton of time to implement)